staging/src/github.com/openshift/oc/pkg/helpers/image/test/util.go

```diff
@@ -33,6 +32,10 @@ var (
 	Config2 = "sha256:8ddc19f16526912237dd8af81971d5e4dd0587907234be2b83e249518d5b673f"
 )
 
+const (
+	managedByOpenShiftAnnotation = "openshift.io/image.managed"
```

Comment:
move to openshift/api and replace here.  You can have it as a followup don't forget.

Reply:
@deads2k done in https://github.com/openshift/api/pull/338